modules/python/tests/test_python_template.c

```diff
@@ -53,7 +53,9 @@ _py_init_interpreter(void)
   }
   PyGILState_Release(gstate);
 
+#if PY_MAJOR_VERSION == 2 || (PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION < 7)
   PyEval_InitThreads();
+#endif
```

Comment:
This might be extracted into the python compat code so we wouldn't have these ugly ifdefs.

Reply:
Good point, I was just quickly throwing stuff together to see a green light on github actions side. :)

Reply:
Done. Thanks for pointing out the compat layer. :)